.travis.yml

```diff
@@ -45,6 +45,12 @@ matrix:
     - php: 7.3
       env:
         - DEPS=latest
+    - php: 7.4
+      env:
+        - DEPS=lowest
```

Comment:
Lowest can not be used on 7.4 at this time as the phpunit below 7 will cause deprecation warnings.
One option is to do `composer update phpunit/phpunit --update-with-dependencies`, however in other repositories we opted to omit 7.4 lowest until unsupported php versions are dropped and phpunit bumped to 8.